docs/getting-started/prerequisites.md

```diff
@@ -285,6 +285,10 @@ Alternatively, you can install Helm Diff by following the instructions below.
 
 ### OpenJDK
 
+!!! tip
+    In case you have trouble installing the right jdk version,
```

Comment:
Not sure if people are actually having _trouble_ installing the right JDK. I would reformulate this as follows:

> For easy installation and management of parallel JDK versions check [sdkman](https://sdkman.io/).

Reply:
It depends on the distro : )